feat: cocoapods ide instructions (#4429)
JetBrains/kotlin-web-site: docs/topics/native/native-cocoapods.md

If you encounter problems during the installation, check the [Possible issues and solutions](#possible-issues-and-solutions) section.

## Create a project

> If your environment is set up correctly, you can create a new Kotlin Multiplatform project. For that, use the

Reviewer: ```suggestion
When your environment is set up correctly, you can create a new Kotlin Multiplatform project. For that, use the
```

More optimistic :)

Kotlin Multiplatform web wizard or the Kotlin Multiplatform plugin for Android Studio.

### Using web wizard

To create a project using the web wizard and configure the CocoaPods integration:

> 1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com) and select target platforms for your project.

Author: I think it's capitalized there because of the headline-style capitalization used across marketing materials. For me, here it's just a common noun, like 'editor'

2. Click the **Download** button and unpack the downloaded archive.
3. In Android Studio, click **Open** on the Welcome screen or select **File | Open** in the editor.
4. Navigate to the unpacked project folder and then click **Open**.
5. Add the Kotlin CocoaPods Gradle plugin to the version catalog. In the `gradle/libs.versions.toml` file,
   add the following declaration to the `[plugins]` block:
 
   ```toml
   kotlinCocoapods = { id = "org.jetbrains.kotlin.native.cocoapods", version.ref = "kotlin" }
   ```
   
6. Navigate to the root `build.gradle.kts` file of your project and add the following alias to the `plugins {}` block:

   ```kotlin
   alias(libs.plugins.kotlinCocoapods) apply false
   ```

7. Open the module where you want to integrate CocoaPods, for example the `composeApp` module, and add the following alias
   to the `plugins {}` block:

   ```kotlin
   alias(libs.plugins.kotlinCocoapods)
   ```

Now, you are ready to use CocoaPods in your Kotlin Multiplatform project.

### In Android Studio

To create a project in Android Studio with the CocoaPods integration:

> 1. In Android Studio, search for the [Kotlin Multiplatform plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform)

Author: Right, for an advanced topic like this, plugin installation should be easy without details.

   in the Marketplace and install it.

> 2. In Android Studio, click **New Project** on the Welcome screen or select  **File** | **New** | **New Project** in the editor.

Reviewer: And by the way, I think we can cut out the reference to the welcome screen altogether — the same "New project main" menu item is available there, and it's more reliable than a welcome screen whose layout may change unexpectedly.

> 2. In Android Studio, click **New Project** on the Welcome screen or select  **File** | **New** | **New Project** in the editor.

Author: True, let's keep it short, removed the welcome screen option

3. In the list of project templates, select **Kotlin Multiplatform App** and then click **Next**.
4. Name your application and click **Next**.
5. Choose **CocoaPods Dependency Manager** as the iOS framework distribution option.

>    ![AS wizard with the Kotlin Multiplatform plugin](as-project-wizard.png){width=700}

Reviewer: ```suggestion
   ![Android Studio wizard with the Kotlin Multiplatform plugin](as-project-wizard.png){width=700}
```

As this is text for screen-readers and doesn't take space on the page, I'd say avoid acronyms which may not be immediately obvious out loud.

6. Keep all other options default. Click **Finish**.

>    The plugin will automatically generate the project with CocoaPods integration setup.

Reviewer: ```suggestion
   The plugin will automatically generate the project with the CocoaPods integration set up.
```

"Setup" can also work grammatically, but we can use less nouns like this.
